app/models/inbox.rb

```diff
@@ -33,7 +33,10 @@ class Inbox < ApplicationRecord
   include Avatarable
   include OutOfOffisable
 
-  validates :name, presence: true, format: { with: /^\b[\w\s]*\b$/, multiline: true }
+  # Not allowing characters:
+  validates :name, presence: true,
+                   format: { with: %r{^^\b[^/\\<>@]*\b$}, multiline: true,
+                             message: 'Should not start and end with symbols and should not have < > / \ @ symbols' }
```

Comment:
```suggestion
                             message: 'The name should not start or end with symbols, and it should not have < > / \ @ characters.' }
```

Reply:
@pranavrajs Name is auto included in the validation message, as it is the attribute name.